src/routes/scout/[team_data]/+page.svelte

```diff
@@ -1,46 +1,104 @@
 <script lang="ts">
-	import type { TeamMatch, AutoActionData } from '$lib/types';
+	import type { TeamMatch, AutoActionData, AutoHeldItems, TeleActionData } from '$lib/types';
 	import Timeline from '$lib/components/Timeline.svelte';
-	import ActionInputs from './ActionInputs.svelte';
+	import AutoActionInputs from './AutoActionInputs.svelte';
+	import TeleActionInputs from './TeleActionInputs.svelte';
 	import type { PageData } from './$types';
+	import { ArrowRight, ArrowLeft } from 'lucide-svelte';
 	import { browser } from '$app/environment';
 
 	const { data }: { data: PageData } = $props();
 
 	const scout_id = browser ? (localStorage?.getItem('scout_id') ?? '') : '';
 
 	let actions: AutoActionData[] = $state([]);
+	let held: AutoHeldItems = $state({
+		bunnies: 0,
+		balloons: 0,
+		totes: 0
+	});
+	// The furthest index in actions that was made during auto
+	let furthest_auto_index = $state(0);
+
+	let skill = $state(0);
+	let broke = $state(false);
+	let died = $state(false);
+	let notes = $state('');
+
 	let timelineExtended = $state(false);
+	let gamePhase = $state('Auto') as 'Auto' | 'Tele' | 'Post';
 	let pageName = $state('');
 
-	const match: TeamMatch = $state({
-		id: 0,
-		scout_id,
-		team_key: data.team_key,
-		match_key: data.match_key,
-		skill: 3,
-		notes: '',
-		broke: false,
-		died: false,
-		auto_actions: [],
-		tele_actions: []
-	});
+	function phaseShiftRight() {
+		gamePhase = gamePhase === 'Auto' ? 'Tele' : gamePhase === 'Tele' ? 'Post' : 'Post'; // Last case should never happen
+	}
+	function phaseShiftLeft() {
+		gamePhase = gamePhase === 'Post' ? 'Tele' : gamePhase === 'Tele' ? 'Auto' : 'Auto'; // Last case should never happen
+	}
+
+	function submit() {
+		const auto_actions = actions.slice(0, furthest_auto_index + 1);
+		const tele_actions = actions.slice(furthest_auto_index + 1) as TeleActionData[]; // TODO: Add verification function to ensure that this always works
+		const match: TeamMatch = {
+			id: 0,
+			scout_id,
+			team_key: data.team_key,
+			match_key: data.match_key,
+			skill,
+			broke,
+			died,
+			notes,
+			auto_actions,
+			tele_actions
+		};
+	}
 </script>
 
 <div class="m-auto flex h-dvh max-w-md flex-col items-center gap-2 p-2">
 	<div class="flex w-full justify-between border-b-2 border-white/10 pb-2 font-semibold">
-		<span class="flex-grow">Team {data.team_key}</span>
-		<span class="flex-grow text-right">{pageName}</span>
+		<span class="">Team {data.team_key}</span>
+		{#if gamePhase !== 'Auto'}
+			<button onclick={phaseShiftLeft} class=""><ArrowLeft /></button>
```

Comment:
would be better to have the arrows always visible and just disable them instead of removing them b/c it makes the ui move around too much